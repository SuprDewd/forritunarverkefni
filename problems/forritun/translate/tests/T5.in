funny=fyndið
this=þetta
is=er
lol=haha
text:
lol this is funny lol lol
