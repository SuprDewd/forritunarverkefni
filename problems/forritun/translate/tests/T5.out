haha Ã¾etta er fyndiÃ° haha haha
